### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by redodin2**

### CODE REVIEW

File: src/SyncMe/SyncMe.Common/ISyncEventsRepository.cs [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-    Guid AddSyncEvent(SyncEvent syncEvent);
-    void RemoveEvents(Func<SyncEvent, bool> predicate);
+    SyncEvent AddSyncEvent(SyncEvent syncEvent);
+    void RemoveEvents(Expression<Func<SyncEvent, bool>> predicate);
+    void RemoveEvent(Guid eventId);
+    IReadOnlyCollection<SyncEvent> GetByNamespace(string namespaceKey);
+    void UpdateEvents(IEnumerable<SyncEvent> syncEvents);
+    IReadOnlyCollection<SyncEvent> GetNotRepeatableEvents(SyncEventQuery query);
+    IReadOnlyCollection<SyncEvent> GetEveryYearRepeatableEvents(SyncEventQuery query);
+    IReadOnlyCollection<SyncEvent> GetRepeatableLessMonthEvents(SyncEventQuery query);
```
Comment (imkwaq) [inline src/SyncMe/SyncMe.Common/ISyncEventsRepository.cs:17]: What's the intent of this method? A bit strange name

File: src/SyncMe/SyncMe.Common/Models/ConfigKey.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace SyncMe.Models;
+
+public enum ConfigKey
+{
+    IsMute = 1,
+    IsDarkTheme = 2
```
Comment (imkwaq) [inline src/SyncMe/SyncMe.Common/Models/ConfigKey.cs:6]: Do we still need this one?

**Review by imkwaq — APPROVED** [review]
Only minor notes, approved

File: src/SyncMe/SyncMe.Common/ISyncEventsRepository.cs (hunk 1 above)
Comment (redodin2) [reply src/SyncMe/SyncMe.Common/ISyncEventsRepository.cs:17]: I know, I'd like to refactor it to divide all types of Repeatability of events. The meaning of this name is repeatability frequency less than month

File: src/SyncMe/SyncMe.Common/Models/ConfigKey.cs (hunk 2 above)
Comment (redodin2) [reply src/SyncMe/SyncMe.Common/Models/ConfigKey.cs:6]: Let's create a new task to remove all `Dark Theme` related code including icons

**Pull request merged**